addons/website/views/snippets/s_contact_info.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_contact_info" name="Contact Info">
+    <section class="s_contact_info pt48 pb64 o_cc o_cc1">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="11">
+                <div class="o_grid_item g-col-lg-6 g-height-5 col-lg-6 col-10 offset-1 order-lg-0" style="order: 2;--grid-item-padding-x: 0px; --grid-item-padding-y: 24px; grid-area: 1 / 1 / 6 / 7; z-index: 1;">
+                    <h2>Contact Us</h2>
+                    <p class="lead">We'd love to hear from you! If you have any questions, feedback, or need assistance, please feel free to reach out to us using the contact details provided. Our team is here to help and will respond as soon as possible. Thank you for getting in touch!</p>
+                </div>
+                <div class="o_grid_item g-col-lg-6 g-height-2 col-lg-6 col-10 offset-1 order-lg-0" style="order: 3; --grid-item-padding-x: 0px; --grid-item-padding-y: 16px; grid-area: 6 / 1 / 8 / 7; z-index: 2;">
+                    <h3 class="h5-fs">
+                        <i class="fa fa-fw fa-envelope" role="presentation"/>
+                        Email
+                    </h3>
+                    <p>
+                        &#160;&#160;&#160;&#160;&#160;&#160;&#160;<a href="mail-to:info@yourcompany.example.com" title="Send an email">info@yourcompany.example.com</a>
+                    </p>
+                </div>
+                <div class="o_grid_item g-col-lg-6 g-height-2 col-lg-6 col-10 offset-1 order-lg-0" style="order: 4; --grid-item-padding-x: 0px; --grid-item-padding-y: 16px; grid-area: 8 / 1 / 10 / 7; z-index: 3;">
+                    <h3 class="h5-fs">
+                        <i class="fa fa-fw fa-phone" role="presentation"/>
+                        Phone
+                    </h3>
+                    <p>
+                        &#160;&#160;&#160;&#160;&#160;&#160;&#160;<a href="tel:+1555-555-5556" title="Call Customer Service">+1555-555-5556</a>
+                    </p>
+                </div>
+                <div class="o_grid_item g-col-lg-6 g-height-2 col-lg-6 col-10 offset-1 order-lg-0" style="order: 5; --grid-item-padding-x: 0px; --grid-item-padding-y: 16px; grid-area: 10 / 1 / 12 / 7; z-index: 4;">
+                    <h3 class="h5-fs">
+                        <i class="fa fa-fw fa-map-marker" role="presentation"/>
```

Comment:
Prefer the use of `-o` icons ;)
![image](https://github.com/user-attachments/assets/c2be2bf8-4c02-4c46-8540-5191d7b50deb)